docker/build-image.sh

```diff
@@ -54,18 +60,35 @@ fi
 echo "Building docker image for $IMAGE version: $COMPONENT_VERSION_STRING"
 echo "Final image name: $USERNAME/$IMAGE:$COMPONENT_VERSION_STRING"
 
+# Get arguments for ssc_pm_lexus
+if [ -z $access_id ];
+    then 
+        echo "No argument provided for access_id, this script needs to be run with <ACCESS_ID> <SECRET_KEY>"
+        exit 1
+fi
+
+if [ -z $secret_key ];
+    then 
+        echo "No argument provided for secret_key, this script needs to be run with <ACCESS_ID> <SECRET_KEY>"
+        exit 1
+fi
```

Comment:
The way it is currently set up the image would not fail to build. It would just not install the ssc package. But having this makes sense, i'll set this up to happen first before we start building the packages